### ISSUE TITLE
Schedulers: Adding Splay: to a schedule set in minutes causes an infinite loop

### ISSUE THREAD

**Issue by tn8or** [saltstack/salt#32712]
When trying to set up a Schedule for doing state.apply every 30 minutes, I came across a bit of a bug, it seems.
I wanted to add a bit of Splay to my setup, but this resulted in Salt inadvertently adding in a "seconds: 0" config in the job - causing the job to restart as soon as it finishes.

### Setup
State:
```
highstate-schedule:
  schedule.present:
    - function: state.apply
    - minutes: 30
    - splay: 5
```
Results in schedule:
```
    schedule:
      highstate-schedule:
        enabled: true
        function: state.apply
        jid_include: true
        maxrunning: 1
        minutes: 30
        name: highstate-schedule
        seconds: 0
        splay: 5
```
... where the "seconds: 0" config line makes it restart as soon as it's finished a run.

Changing it to:
```
highstate-schedule:
  schedule.present:
    - function: state.apply
    - seconds: 1800
    - splay: 30
```
Fixes up the Schedule:
```
      highstate-schedule:
        enabled: true
        function: state.apply
        jid_include: true
        maxrunning: 1
        name: highstate-schedule
        seconds: 1800
        splay: 30
```
... which works as expected. :-)

### Versions Report
```
# salt --versions-report

Salt Version:
           Salt: 2015.8.8.2

Dependency Versions:
         Jinja2: 2.7.2
       M2Crypto: Not Installed
           Mako: 0.9.1
         PyYAML: 3.10
          PyZMQ: 14.0.1
         Python: 2.7.6 (default, Jun 22 2015, 17:58:13)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.4
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: 1.5
          gitdb: 0.5.4
      gitpython: 0.3.2 RC1
          ioflo: Not Installed
        libgit2: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.3.0
   mysql-python: 1.2.3
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: 0.8.2
        timelib: Not Installed

System Versions:
           dist: Ubuntu 14.04 trusty
        machine: x86_64
        release: 4.2.0-27-generic
         system: Ubuntu 14.04 trusty

```

@behni4
@Inveracity

**Comment by garethgreenaway** [comment]
Thought I had fixed this but it keeps popping up.  Will take a look.

**Comment by garethgreenaway** [comment]
Looking at the latest 2015.8 from GIT, I'm not seeing the issue, so I believe it's fixed in the latest 2015.8.  If you're using packages they might be a bit behind latest git.

**Comment by jfindlay** [comment]
@tn8or, thanks for reporting, and @garethgreenaway, thanks for looking into this.
